### ISSUE TITLE
tls_private_key fails to plan cleanly after import

### ISSUE THREAD

**Issue by dekimsey** [hashicorp/terraform-provider-tls#283]
### Terraform CLI and Provider Versions

Terraform v1.3.1
on darwin_arm64
+ provider registry.terraform.io/hashicorp/tls v3.4.0

### Terraform Configuration

```terraform
resource "tls_private_key" "demo" {
    algorithm = "ECDSA"
    ecdsa_curve = "P256"
}
```

### Expected Behavior

Clean apply after import

### Actual Behavior

Plan shows a forced replacement due to the ecdsa_curve not round-tripping after import. Unfortunately this is a destructive action.

```
Terraform will perform the following actions:

  # tls_private_key.ca must be replaced
-/+ resource "tls_private_key" "ca" {
      ~ ecdsa_curve                   = "P-256" -> "P256" # forces replacement
...
```

### Steps to Reproduce

1. `openssl ecparam -name prime256v1 -genkey -noout -out demo.key`
2. `terraform import tls_private_key.demo ./key.pem`
3. `terraform plan -detailed-exitcode`

### How much impact is this issue causing?

Low

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bflad** [comment]
Hi @dekimsey 👋 

[Official releases of the hashicorp/tls provider](https://registry.terraform.io/providers/hashicorp/tls) do not include support for importing the `tls_private_key` resource:

Version 3.4.0:

```console
$ terraform import tls_private_key.test demo.key 
tls_private_key.test: Importing from ID "demo.key"...
╷
│ Error: resource tls_private_key doesn't support import
```

Version 4.0.3 (latest as of writing):

```console
$ terraform import tls_private_key.test demo.key
tls_private_key.test: Importing from ID "demo.key"...
╷
│ Error: Resource Import Not Implemented
│ 
│ This resource does not support import. Please contact the provider developer for additional information.
```

There is, however, an open feature request for adding this functionality available at #44. Since this issue appears to represent unofficial functionality and there's already an associated issue for tracking the enhancement, I'm going to close this. Thanks.

**Comment by dekimsey** [comment]
No worries, thank you. I wasn't sure if I should have commented but I didn't want to derail the other ticket. And the feature appeared to exist if not documented so I wasn't sure what to make of it's state.
